acceptance/acceptance_test.go

```diff
@@ -0,0 +1,290 @@
+package acceptance_test
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"maps"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"slices"
+	"sort"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/databricks/cli/internal/testutil"
+	"github.com/databricks/cli/libs/env"
+	"github.com/databricks/cli/libs/testdiff"
+	"github.com/stretchr/testify/require"
+)
+
+var KeepTmp = os.Getenv("KEEP_TMP") != ""
+
+func TestAccept(t *testing.T) {
+	execPath := BuildCLI(t)
+	t.Setenv("CLI", execPath)
+
+	server := StartServer(t)
```

Comment:
By using test server we bypass all the auth related code comparing to existing integration tests where we don't, do you plan to allow to run these tests without test server in some way?